clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -502,6 +502,12 @@ Changes in existing checks
   usages of ``std::string_view::compare``. Added a `StringLikeClasses` option
   to detect usages of ``compare`` method in custom string-like classes.
 
+- Improved :doc:`readability-redundant-smartptr-get
+  <clang-tidy/checks/readability/readability-redundant-smartptr-get>` identify
```

Comment:
duplicated readability, put check list in alphabetical order. and instead of duplicating check description, put information what were changed/fixed in check behavior.

Reply:
I did not understand. Can you explain how it's needs to be written?

Reply:
you got "readability/readability-redundant...." when should be "readability/redundant...."

Reply:
and remove this entry, leave only one that you added at line 496.

Reply:
and this change log does not describe what you changes, and should teel about improved fixits and -> leftover.